includes/wc-admin-update-functions.php

```diff
@@ -316,3 +316,42 @@ function wc_admin_update_290_update_apperance_task_option() {
 function wc_admin_update_290_db_version() {
 	Installer::update_db_version( '2.9.0' );
 }
+
+/**
+ * Use woocommerce_admin_activity_panel_inbox_last_read from the user meta to set wc_admin_notes.is_read col.
+ */
+function wc_admin_update_300_update_is_read_from_last_read() {
```

Comment:
Should we clean up the `woocommerce_admin_activity_panel_inbox_last_read` user meta option and delete after `is_read` has been updated?

Reply:
That's a good idea 👍